src/components/presentation/TrendingArticles.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import './TrendingArticles.scss';
+
+const defaultImage = process.env.REACT_APP_DEFAULT_ARTICLE_IMAGE;
+
+const TrendingArticles = ({ articles }) => {
+  const [topTrending, ...otherTrending] = articles;
+  return (
+    <section className="trends">
+      <h1 className="heading">Trending articles</h1>
+      <div className="row">
+        <div className="col s12 m6">
+          <article className="trend top">
+            <img src={topTrending.imgUrl || defaultImage} alt={topTrending.title} />
+            <h3 className="title truncate"><Link to={`/articles/${topTrending.slug}`}>{topTrending.title}</Link></h3>
+            <p className="description truncate">{topTrending.description}</p>
+            <p className="author">
+              {'by '}
+              <span>{topTrending.author.username}</span>
+            </p>
+            <div className="more">
```

Comment:
The actual rating component would make sense here since all articles coming from the API currently have a rating property. But I get that since this current implementation serves like somewhat of a placeholder.